src/bridge/ui_commands.rs

```diff
@@ -276,15 +278,15 @@ impl AsRef<str> for UiCommand {
 }
 
 struct UIChannels {
-    sender: LoggingSender<UiCommand>,
+    sender: Mutex<Option<LoggingSender<UiCommand>>>,
```

Comment:
These Mutex Options over concurrent capable types kill me a little, but after thinking about it for a bit nothing cleaner came to mind so I think its reasonable to leave it.

One thing we might consider is to use a parkinglot RwLock instead since the base case is just reading and we only write when the app closes. But tbh thats a micro optimization to take it or leave it

Reply:
I simplified this, using `OnceLock`